partners.html

```diff
@@ -0,0 +1,100 @@
+</head>
+<body>
+  <header class="header-container" >
+    <div class="first-section-nav">
+      <ul class="section-nav-list">
+        <li ><img class="section-nav-item" src='/assets/facebook.png' alt="logo"></li>
+        <li ><img class="section-nav-item" src='/assets/twitter.png' alt="logo"></li>
+        <li>French</li>
+        <li>English</li>
+        <li>Log out</li>
+      </ul>
+    </div>
+    <div class="header">
+      <button class="moile-menu-hamburger">
+        <span class="hamburger"></span>
+      </button>
+      <h2 class="logo-uncc"> <a href="/index.html">UNCC</a></h2>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item"><a href="/partners.html">About</a></li>
+          <li class="nav-item"><a href="#" class="black">Program</a></li>
+          <li class="nav-item"><a href="#" class="black">Join</a></li>
+          <li class="nav-item"><a href="#partners" class="black">Partners</a></li>
+          <li class="nav-item"><a href="#" class="black">News</a></li>
+          <li class="nav-item campaign"><a href="#"><p>AT Campaign</p></a></li>
+        </ul>
+      </nav>
+    </div>
+    
+  </header>
+  <main>
+    <section class="section-one">
+      <h1 class="section-one-title">UNITING THE WORLD
+        TO TACKLE CLIMATE CHANGE.
+      </h1>
+      <p class="presentation-description">
+        The COP26 summit brought parties together to accelerate 
+        action towards the goals of the Paris Agreement and the 
+        UN Framework Convention on Climate Change.
+        Learn about how the negotiations at COP26 went and the
+         outcomes achieved in the documents below.
+      </p>
+      <p class="contact-glascow">Please contact us per Email for any further 
+        questions about The Glascow conference. <a href="#">contact@atr.org</a></p>
+    </section>
+    <div class="logo-climat">
+      <h2 class="logo-climat-title border-bottom">CREATIVE  OF  UNCC  LOGO</h2>
+      <p class="logo-climat-detail">UNC Charlotte was relatively unknown outside of the region, 
+        and the administration sought to build a distinctive identity with 
+        the first official logomark. Joe Sonderman, with Jack Pentes Design, 
+        created the block form UNCC logo as a pro bono project.
+      </p>
+      <img class="uncc" src="/assets/UNFCCC-Featured-Image.png" alt="Featured">
+      <!-- <p class="climat-logo-detail">This is the Logo of the UN climat change</p> -->
+    </div>
+    <section class="past-summit">
+      <h2 class="past-summit-title">See the past summit of climat change</h2>
+      <p class="past-summit-detail">Take a look at the last two Editions.</p>
+      <div class="past-summit-project">
+        <div class="past-project-bgr">
+          <h3 class="past-project-title">2013 Summit</h3>
+        </div>
+        <div class="past-project-bgr">
+          <h3 class="past-project-title">2015 Summit</h3>
+        </div>
+      </div>
+    </section>
```

Comment:
I would like to point out some design details for this section:

- [ ] Kindly use `images` and not `background-images` for this section, if an image is a core feature of a project, it is suggested that an `<img>` tag be used but if it is to design the page, `background-image` should be used. 👍 
- [ ] On mobile, the images should be vertically arranged while on desktop, they should be horizontally arranged. 

### Behance Design (Mobile)

![image](https://user-images.githubusercontent.com/77106962/157770890-df46946f-23e4-45b9-a48c-785161e77bf1.png)

### Behance Design (Desktop)

![image](https://user-images.githubusercontent.com/77106962/157770978-7ec1245a-9414-41db-bfb3-46eb8435e979.png)